compiler/incremental-compilation-impl/test/org/jetbrains/kotlin/incremental/classpathDiff/ClasspathChangesComputerTest.kt

```diff
@@ -27,6 +27,9 @@ abstract class ClasspathChangesComputerTest : ClasspathSnapshotTestCommon() {
     @Test
     abstract fun testModifiedAddedRemovedElements()
 
+    @Test
```

Comment:
Can you please add a test for the mixed granularity? E.g:
- class X with previous fine-grained and current coarse-grained
- prev_X(fine) & current_X(fine) and also prev_Y (coarse) & current_Y(coarse)
this is to make sure diffing logic can differentiate the two and that it is also able to sum of changes produced by these 2 different methods.

Reply:
Done - I've added it in the next commit.